packages/nodes-base/package.json

```diff
@@ -14,6 +14,8 @@
     "url": "git+https://github.com/n8n-io/n8n.git"
   },
   "scripts": {
+    "benchmark:build": "tsc -p tsconfig.benchmark.json > /dev/null; tsc-alias -p tsconfig.benchmark.json; pnpm build:metadata",
```

Comment:
why the `> /dev/null` ?

Reply:
To silence type errors when transpiling test helpers. Did not fix those here to avoid a larger PR.